kddeployqt.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
MIT license header?

Reply:
We have the license file, I don't think we need to pollute the src file with licenses.